src/replacements.js

```diff
@@ -0,0 +1,188 @@
+const Gio = imports.gi.Gio;
+const EosGates = imports.eos_gates;
+
+function generateEndlessInstallerEntry() {
+    let entry = {
+        regex: {
+            windows: /endless-installer.*.exe/
+        },
+        overrideHelpMessage: _("You are already running Endless OS")
+    };
+
+    // If we detect the presence of eos-installer.desktop, inform
+    // the user on how to actually install Endless OS.
+    let desktopInfo = Gio.DesktopAppInfo.new('eos-installer.desktop');
+    if (desktopInfo) {
+        entry.overrideHelpMessage = _("You are already running Endless OS from live media");
+        entry.desktopInfo = desktopInfo,
+        entry.replacementInfo = {
+            appName: _("Endless OS Installer"),
+            description: _("You can install Endless OS by running the Endless OS Installer.")
```

Comment:
`eos-installer`, the Linux app, is called "Reformat" on the desktop and "Reformat with Endless OS" in the UI. We try to use "Installer" exclusively to refer to the less-destructive Windows tool, and talk about "Reformat with Endless OS" in this non-dual-boot case. I'd expect:

```
appName: _("Reformat with Endless OS")
description: _("You can reformat with Endless OS by running Reformat")